.travis.yml

```diff
@@ -13,6 +13,8 @@ before_install:
   - export DOCKER_CLI_EXPERIMENTAL=enabled
   - sudo service docker restart
   - docker --version
+  - bash install_requirements.sh
```

Comment:
This install-requirements.sh script is run inside the docker image. There is no need to run it outside, because ambianic-edge itself is tested within the docker image.